rust-toolchain

```diff
@@ -0,0 +1 @@
+1.70.0
```

Comment:
Note that we could keep `rust-toolchain.toml` and instead use the following workaround in workflows:
```
- uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
```
as it ignores the TOML file at first and then runs `rustup show`, which sets the toolchain correctly. However, this is hacky and probably slower so I'm fine either way.

Reply:
Tracking issue for `rust-toolchain.toml` support: https://github.com/actions-rs/toolchain/issues/126
However, the project appears to be unmaintained: https://github.com/actions-rs/toolchain/issues/216, but the popular, maintained alternative https://github.com/dtolnay/rust-toolchain/issues/12 also doesn't support the TOML version.

Reply:
The problem is that this would hardcode stable, which is a bit of a PITA, because we get changes in tooling behavior without a commit to our repo triggering that. Here's an example of this happening in Nova:
https://github.com/microsoft/Nova/issues/178